visualization/app/codeCharta/util/codeMapHelper.ts

```diff
@@ -31,13 +31,29 @@ export function getNodesByGitignorePath(root: CodeMapNode, gitignorePath: string
 	if (gitignorePath.length === 0) {
 		return []
 	}
+	const ignoreResults = returnIgnore(gitignorePath)
+	const filtered = []
+	for (const { data } of hierarchy(root)) {
+		if (ignoreResults.ignoredNodePaths.ignores(transformPath(data.path)) === ignoreResults.condition) {
+			filtered.push(data)
+		}
+	}
+	return filtered
+}
+
+export function IsNodeExcludedOrFlattened(node: CodeMapNode, gitignorePath: string): boolean {
```

Comment:
Functions should start with a lower case :)

Reply:
Thanx will change that.